### PR TITLE
Java install info update

### PR DESCRIPTION
**Pull request by IvyBazan**
## Description ##
Updated dependency information for java per issue #13910. Also, removed duplicated Clojure dependency information

### CODE REVIEW

**Comment by stu1130** [comment]
@mxnet-label-bot add [pr-awaiting-review]
Thanks for your contribution @IvyBazan 
@lanking520 @zachgk @piyushghai @andrewfayres

File: docs/install/index.md [hunk 1]
```diff
@@ -479,9 +479,7 @@ You can use the Maven packages defined in the following dependency to include MX
 <dependency>
     <groupId>org.apache.mxnet</groupId>
     <artifactId>mxnet-full_2.11-linux-x86_64-gpu</artifactId>
-    <scope>system</scope>
-    <version>1.4.0</version>
-    <systemPath>/system/path/to/jar/mxnet-full_2.11-linux-x86_64-gpu-1.4.0-SNAPSHOT.jar</systemPath>
+    <version>LATEST</version>
```
Comment (frankfliu) [inline docs/install/index.md]: LATEST only works for maven2, it doesn't work for maven 3.
It's better to use: https://maven.apache.org/pom.html#Dependency_Version_Requirement_Specification

Comment (piyushghai) [reply docs/install/index.md]: Yes, use something like [1.4.0, )

**Comment by aaronmarkham** [comment]
Fixes #13430

File: docs/install/index.md (hunk 1 above)
Comment (IvyBazan) [reply docs/install/index.md]: @frankfliu Addressed version info in latest commit

**Comment by IvyBazan** [comment]
@mxnet-label-bot add [java]

**Comment by IvyBazan** [comment]
@mxnet-label-bot add [website]

**Pull request merged**